index.html

```diff
@@ -443,6 +443,41 @@ <h2 class="tracks-title">
         <p class="section-title">
           <span class="underline sponsors">sponsors</span>
         </p>
+        <br><br><br>
+
+        <!-- Sponser images -->
+        <div class="container" data-entrance="fade" data-entrance-delay="100">  
+
+        	<div class="row" style="padding: 1em;">
+	    		<div class="col-sm" style="padding: 2em;">
+	    			<img style="object-fit: contain; max-height: 180px;" class="img-fluid logo-picture" src="sponsors/centene.png">
+	    		</div>
+
+	    		<div class="col-sm" style="padding: 2em;">
+	    			<img style="object-fit: contain; max-height: 180px;" class="img-fluid logo-picture" src="sponsors/twilio.png">
```

Comment:
these need to be clickable, so each sponsor needs to link to the site @josharnoldjosh